src/main/java/com/google/sps/servlets/DeleteFolderServlet.java

```diff
@@ -45,7 +45,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
       response.getWriter().println(new Gson().toJson(jsonErrorInfo));
     } else {
       deleteAllCardsInsideFolder(folder);
-      Folder.deleteFolderWithRetries(folder);
+      Folder.deleteFolderWithRetries(folder, /*retries=*/ 5);
```

Comment:
This should usually be `/*retries=*/5`, but don't change if it was the autofomatter that added a space.

Reply:
done